profiles/common/modules/features/ec_embedded_video/includes/ec_embedded_video.formatters.inc

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @file
+ * File formatters for Platform needs.
+ */
+
+/**
+ * Implements hook_file_formatter_info_alter().
+ *
+ * It alters file_formatter_info to add the "Wrapper free formatters for the
+ * different Video-sharing services (YouTube, Vimeo, Dailymotion and
+ * AV portal).
+ */
+function ec_embedded_video_file_formatter_info_alter(&$info) {
```

Comment:
Something like this might help ? http://pastebin.com/C8KyKvGV

Anyway, let's discuss tomorrow. Good night.